aldryn_config.py

```diff
@@ -5,6 +5,7 @@
 
 
 class Form(forms.BaseForm):
+
     enable_debug_toolbar = forms.CheckboxField(
```

Comment:
We should probably remove the field from the form here, and configure from an environment variable instead.